plugin.go

```diff
@@ -27,7 +27,8 @@ type Plugin struct {
 }
 
 func (p *Plugin) Exec(ctx context.Context) error {
-	fmt.Println("Surge.sh preview plugin")
+	fmt.Println("--- Surge.sh preview plugin ---")
+    fmt.Println()
```

Comment:
```suggestion
	fmt.Println("--- Surge.sh preview plugin ---")
        fmt.Println()
```

Reply:
Had this first but this causes a linter warning.

Reply:
Interesting. Ok the keep the two statements, but the indentation loosk off.